### PR TITLE
Iqra calendar performance edit

### PR DESCRIPTION
**Pull request by iqra-javed**
# Description

Please include a summary of the change and a link to which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change
- added ability to edit performance from checkoff performance modal
- refactored calendarPage.js
- fixed some styles such as centering form validation errors and margin top on calendar

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

Tested Locally and on Mlab

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] My changes generate no new warnings

### CODE REVIEW

File: client/src/components/CalendarPage.js [hunk 1]
```diff
@@ -20,64 +19,29 @@ class CalendarPage extends Component {
...
-    return null;
-  }
-
   componentDidMount() {
     this.props.fetchRoutines();
     this.props.fetchAllWorkouts();
-    this.props.fetchAllPerformanceDocs();
   }
 
+  //-------------------------------------------- callback functions for scheduling modal
```
Comment (rusticlion) [inline client/src/components/CalendarPage.js:37]: Good comment. We should spend some time as a group (tomorrow morning)? reading through the codebase together and writing commentary.

File: client/src/components/CalendarPage.js [hunk 2]
```diff
@@ -95,77 +59,137 @@ class CalendarPage extends Component {
...
-    this.props.deleteWorkout(this.selectedEventId);
-    this.checkboxModalToggle();
+  handlePerformanceChange = e => {
+    console.log(e.target);
+    this.setState({ [e.target.name]: e.target.value });
   };
 
   // this updates the toggled boolean of the specified performance doc in the DB
-  handleCheckOffInDB = performanceId => {
+  handlePerformanceUpdate = performanceId => {
```
Comment (rusticlion) [inline client/src/components/CalendarPage.js:75]: Functionality seems to be working like a charm!

**Review by rusticlion — APPROVED** [review]
Feature is working correctly with no apparent bugs introduced. I am thinking about how we could make the calendar more mobile-friendly, but the PR is solid.

**Review by emukupa — APPROVED** [review]
good work

**Pull request merged**